webdriver/conftest.py

```diff
@@ -36,6 +39,25 @@ def session(_session, request):
 
     return _session
 
+
 @pytest.fixture(scope="function")
 def http(session):
     return HTTPRequest(session.transport.host, session.transport.port)
+
+
+@pytest.fixture
+def server_config():
+    return json.loads(os.environ.get("WD_SERVER_CONFIG"))
```

Comment:
No, because that's the wrong host and port… this is the host of the web server, whereas the one above is the host of the webdriver server (one is wptserve and the other is e.g. geckodriver).